railties/lib/rails/tasks/framework.rake

```diff
@@ -64,5 +64,18 @@ namespace :rails do
     task :bin do
       RailsUpdate.invoke_from_app_generator :create_bin_files
     end
+
+    task :schema_migrations => :environment do
+      if ActiveRecord::SchemaMigration.table_exists?
```

Comment:
is there an example of someplace that checks for ActiveRecord presence?
or are you saying something as easy as

```ruby
available = ActiveRecord rescue nil
if available
  ActiveRecord::SchemaMigration.table_name
...
```